tests/unit/metadata/GithubActionMetadata.spec.js

```diff
@@ -0,0 +1,20 @@
+import GithubActionMetadata from 'src/metadata/GithubActionMetadata';
+import { DefaultData } from 'leto-modelizer-plugin-core';
+import fs from 'fs';
+
+describe('Test GithubActionMetadata', () => {
+  describe('Test methods', () => {
+    describe('Test method: parse', () => {
+      it('Default metadata should return valid content', () => {
```

Comment:
suggestion `Should return valid content when parsing default metadata`

Reply:
done